airgun/views/oscapcontent.py

```diff
@@ -0,0 +1,90 @@
+from widgetastic.widget import (
+    FileInput,
+    Text,
+    TextInput,
+    View,
+)
+from widgetastic_patternfly import BreadCrumb
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ActionsDropdown,
+    MultiSelect,
+    SatTable,
+)
+
+
+class SCAPContentsView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='SCAP Contents']")
+    new = Text("//a[contains(@href, 'scap_contents/new')]")
+    table = SatTable(
+        './/table',
+        column_widgets={
+            'Title': Text('./a'),
+            'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
+        }
+    )
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class SCAPContentCreateView(BaseLoggedInView):
+    submit = Text('//input[@name="commit"]')
+    cancel = Text('//a[text()="Cancel"]')
+
+    @View.nested
+    class file_upload(SatTab):
+        TAB_NAME = 'File Upload'
+        title = TextInput(id='scap_content_title')
+        scap_file = FileInput(id='scap_content_scap_file')
+
+    @View.nested
+    class locations(SatTab):
+        resources = MultiSelect(id='ms-scap_content_location_ids')
+
+    @View.nested
+    class organizations(SatTab):
+        resources = MultiSelect(id='ms-scap_content_organization_ids')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
```

Comment:
we can refer to element `<form class="form-horizontal well" id="new_scap_content" enctype="multipart/form-data" action="/compliance/scap_contents" accept-charset="UTF-8" method="post"><div class="fr close-button"><span class="pficon pficon-close two-pane-close"></span></div><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="Wc8Zjc+0VjHSmI+CbiPbDzJrz/MjIMShx5VzYaoKY1lUqd+eBquZ1tHsW4r7DosLMziVMIfw67Of5NEXE5tp7g==">
```
as edit also have `title = TextInput(id='scap_content_title')`, so it is ambigous

Reply:
you can check
create_form = Text(//form[@id='new_scap_content'])
def is_displayed(self):
    return self.browser.wait_for_element(self.create_form , exception=False) is not None

```